Server/JokesOnYou.Web.Api/Data/DataSeeding.cs

```diff
@@ -15,18 +15,20 @@ public static void Initialize(DataContext context)
         {
             context.Database.EnsureCreated(); //Make sure db actually exists
 
-            // Look for any students.
+            // Look for any users in db
             if (context.Users.Any())
             {
+                //If there are students then leave method
                 return;
             }
 
-            var jsonThingy = JsonConvert.DeserializeObject<List<User>>(File.ReadAllText(AppContext.BaseDirectory + "SeedingDB.json"));
+            //If you go here than it means that the db is empty
+            var TemporaryUsers = JsonSerializer.Deserialize<List<User>>(File.ReadAllText(@"Data\UserSeedData.json"));
```

Comment:
should be camelcase
i would vote for it being named just users since its a list of users. Or usersToAdd

Reply:
Oh, yeah should fix that, whoops